libdnf/dnf-repo.cpp

```diff
@@ -973,6 +973,25 @@ dnf_repo_set_keyfile_data(DnfRepo *repo, GError **error)
 
     /* baseurl is optional; if missing, unset it */
     baseurls = g_key_file_get_string_list(priv->keyfile, repoId, "baseurl", NULL, NULL);
+    if (baseurls) {
+        // baseruls can be ['value1', 'value2, value3'] therefore we first join to have 'value1, value2, value3'
+        tmp_strval = g_strjoinv(",", baseurls);
+        auto & repoBaseurls = priv->repo->getConfig()->baseurl();
+        repoBaseurls.set(libdnf::Option::Priority::REPOCONFIG, "");
```

Comment:
Why? Is it needed to set "" before `bindBaseurls.newString()` is called?

Reply:
I have removed it, before I didn't check how the config options work properly and I just copied it from excludepkgs code. Here it is not necessary since we are not combining two options like the `exclude` and ` excludepkgs` do.